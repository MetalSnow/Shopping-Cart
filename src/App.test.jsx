import { describe, it, expect } from 'vitest';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import App from './App';

describe('App Component', () => {
  it('Render Homepage', () => {
    const { container } = render(
      <MemoryRouter>
        <App />
      </MemoryRouter>
    );

    expect(container).toMatchSnapshot();
  });
});
