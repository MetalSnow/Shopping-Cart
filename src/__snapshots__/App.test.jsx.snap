// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App Component > Render Homepage 1`] = `
<div>
  <div
    class="_container_2aae8b"
  >
    <a
      href="/"
    >
      <h1
        class="_title_2aae8b"
      >
        <svg
          class="lucide lucide-store"
          fill="none"
          height="40"
          stroke="white"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="40"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="m2 7 4.41-4.41A2 2 0 0 1 7.83 2h8.34a2 2 0 0 1 1.42.59L22 7"
          />
          <path
            d="M4 12v8a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2v-8"
          />
          <path
            d="M15 22v-4a2 2 0 0 0-2-2h-2a2 2 0 0 0-2 2v4"
          />
          <path
            d="M2 7h20"
          />
          <path
            d="M22 7v3a2 2 0 0 1-2 2a2.7 2.7 0 0 1-1.59-.63.7.7 0 0 0-.82 0A2.7 2.7 0 0 1 16 12a2.7 2.7 0 0 1-1.59-.63.7.7 0 0 0-.82 0A2.7 2.7 0 0 1 12 12a2.7 2.7 0 0 1-1.59-.63.7.7 0 0 0-.82 0A2.7 2.7 0 0 1 8 12a2.7 2.7 0 0 1-1.59-.63.7.7 0 0 0-.82 0A2.7 2.7 0 0 1 4 12a2 2 0 0 1-2-2V7"
          />
        </svg>
        Demo Store
      </h1>
    </a>
    <ul
      class="_listContainer_2aae8b"
    >
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/shop"
        >
          Shop
        </a>
      </li>
      <li>
        <a
          href="https://github.com/MetalSnow"
          target="_blank"
        >
          About
        </a>
      </li>
    </ul>
  </div>
  <div
    class="_container_148466"
  >
    <div>
      <p>
        Refresh your wardrobe with our latest collection of everyday essentials!
      </p>
      <img
        alt="Shop-pics"
        src="/src/assets/home-pic-one.jpg"
      />
    </div>
    <div>
      <img
        alt="Shop-pics"
        src="/src/assets/home-pic-two.jpg"
      />
      <p>
        Experience ultimate comfort with our stylish casual wearâ€”perfect for any occasion!
      </p>
    </div>
    <div>
      <p>
        Step out in confidence with our fashionable statement pieces that elevate any outfit!
      </p>
      <img
        alt="Shop-pics"
        src="/src/assets/home-pic-three.jpg"
      />
    </div>
  </div>
</div>
`;
